specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/examples/expressRouteProviderPortList.json

```diff
@@ -0,0 +1,48 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "portPairDescriptor": "bvtazureixpportpair1",
+            "id": "/subscriptions/subid/providers/Microsoft.Network/ExpressRoutePortsLocations/SiliconValley/bvtazureixpportpair1",
```

Comment:
The IDs in these are not correct.  The API is listing expressRouteProviderPorts.  The location naming should not be in this ID either

Reply:
can you please tell me what else is wrong apart from location? I forgot to remove the location when I added the filter.